styles.css

```diff
@@ -85,4 +87,185 @@
   .social-icons li {
     margin: 0 10px;
   }
+
+  .list-item {
+    background-color: #ebf0ee;
+    border: 1px solid #d0d9d4;
+    border-radius: 5px;
+    width: 90%;
+    margin-top: 10px;
```

Comment:
- [ ] Kindly use `flexbox` to lay out all individual cards of the `work` section as stated in the project requirement 🙏 .